util/os-runner/install-dmd.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-wget https://netcologne.dl.sourceforge.net/project/d-apt/files/d-apt.list -O /etc/apt/sources.list.d/d-apt.list
-apt-get update --allow-insecure-repositories
-apt-get -y --allow-unauthenticated install --reinstall d-apt-keyring
-apt-get update && sudo apt-get install -y dmd-compiler dub
+curl -fsS https://dlang.org/install.sh | bash -s dmd
+OLD_PS1=$PS1
+source "$(find ~/dlang -maxdepth 1 -name 'dmd-*')/activate"
+PS1=$OLD_PS1
```

Comment:
```suggestion
source "$(find ~/dlang -maxdepth 1 -name 'dmd-*')/activate"
```
Changes to `PS1` are useless, it is set by `config-terminal.sh` after DMD is installed.